next.clayui.com/content/docs/css/components/sidebar.md

```diff
@@ -0,0 +1,470 @@
+---
+title: "Sidebar"
+description: "A vertical panel that appears from the right or the left side of the screen, triggered by a button."
+---
+
+<div class="nav-toc">
+
+- [Structure](#structure)
+- [Variations](#variation)
+    - [Light](#light)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/sidebar/">Sidebar Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+## Structure
+
+Sidebar is an opinionated container to display related content.
+
+Is composed by:
+
+```html
+<div class="sidebar-header">
+    Sidebar Header
+</div>
+```
+
+```html
+<div class="sidebar-body">
+    Sidebar Body
+</div>
+```
+
+```html
+<div class="sidebar-footer">
+    Sidebar Footer
+</div>
+```
+
+## Variations
+
+### Sidebar Light
+
+Just add `sidebar-light` class on the same element that you are using `sidebar`.
+
+<div class="sheet-example">
+    <div class="sidebar sidebar-light">
```

Comment:
@diegonvs would be better to wrap it with a `.row` and `.col-md-6` for the example not to get so wide and look like it does not follow Lexicon examples.

```html
<div class="row">
  <div class="col-md-6">
    ...
  </div>
</div>
```